migrations/educandu-2022-06-07-01-add-question-and-answers-to-interactive-media-chapters.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase, no-console */
+export default class Educandu_2022_06_07_01_add_question_and_answers_to_interactive_media_chapters {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async collectionUp(collectionName) {
+    const result = await this.db.collection(collectionName).updateMany(
+      {},
+      {
+        $set: {
+          'sections.$[sectionElement].content.chapters.$[].question': '[Question]',
+          'sections.$[sectionElement].content.chapters.$[].answers': ['[Answer]'],
+          'sections.$[sectionElement].content.chapters.$[].correctAnswer': 0
```

Comment:
Maybe `correctAnswerIndex` would be even better?

Reply:
Yes, clearer, renamed!